cinema/views.py

```diff
@@ -167,6 +193,24 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type=str,
+                description="date in format year-month-day",
+                required=False,
+            ),
+            OpenApiParameter(
+                "movie",
+                type=int,
+                description="movie id"
```

Comment:
The description for the 'movie' parameter in the OpenApiParameter should refer to 'movie id' instead of 'genres'. This seems to be a copy-paste error and should be corrected to accurately describe the parameter being used.